plugins/synaptics-mst/synaptics-mst.quirk

```diff
@@ -40,7 +40,8 @@ Name = Dell Rugged Platform
 DeviceKind = system
 
 [SynapticsMSTBoardID=259]
-Name = Dell dock
+Name = Dell WD19 dock
+Plugin = dell_dock
```

Comment:
I feel like this chunk should probably live in the dell-dock quirk file now.

Reply:
Actually, maybe we just remove the chunk altogether? Or do something like Flags=ignore?